### ISSUE TITLE
player can't join server

### ISSUE THREAD

**Issue by TEELEKza25** [VeroFess/PalWorld-Server-Unoffical-Fix#18]
![image](https://github.com/VeroFess/PalWorld-Server-Unoffical-Fix/assets/157742568/8c74b973-8ea5-4441-8c22-c9a4a5dcf34a)

sometime some player can't joins server got to restart server then some player can join but some still got a problem
is this related to your unofficial fix? or can you fix it
谢谢你

**Comment by Kotori0629** [comment]
maybe network issue?

**Comment by xxlio109** [comment]
my server is also experiencing this problem with server. Win64, you need to restart the server to join

**Comment by LukasPob** [comment]
it is infinite loading screen issue, i think its unrelated to this as ive never used it and 2 of my servers have it

**Comment by Re-Aqest** [comment]
This is the official bug, waiting for the official repair.

**Comment by VeroFess** [comment]
This seems to require modifications to both the server and the client. If the official next major version does not fix this problem, I will try to fix it.

**Comment by karlseven** [comment]
what i observed is that when your Level.sav reached more than 20mb+ people started to have this infinite loop.

if its below 20mb everything is good

**Comment by etreee** [comment]
> what i observed is that when your Level.sav reached more than 20mb+ people started to have this infinite loop.
> 
> if its below 20mb everything is good

Running into this issue as well. Though before the 1.3.0 update we were at 27mb and had no issues. Also noted that the server stops responding to RCON commands after being online for a while. Definitely something that was introduced with the official 1.3.0 client and server update.

**Comment by VeroFess** [comment]
Since the official debugging information has been removed, it‘s very difficult to modify binary correctly - I mean it may look fixed, but may introduce more potential problems.

Since this seems to be a problem that the official will actively fix, I still recommend waiting for the official fix rather than taking the risk of doing it ourselves.

**Comment by karlseven** [comment]
> Since the official debugging information has been removed, it‘s very difficult to modify binary correctly - I mean it may look fixed, but may introduce more potential problems.
> 
> Since this seems to be a problem that the official will actively fix, I still recommend waiting for the official fix rather than taking the risk of doing it ourselves.

Thank you for doing this. actually your patched file helped somehow. better game performance with 32 users online. i do hope you continue to fix whatever you can fix while we wait for official patches

**Comment by LukasPob** [comment]
Guys that get thsi issue, could you check if you have auto guild destruct enabled?

**Comment by etreee** [comment]
> Guys that get thsi issue, could you check if you have auto guild destruct enabled?

Auto guild destruction is disabled for me and I was still encountering this earlier today.

Did run into a new issue on this though, but it's not related to the fix. Level.sav reached 34MB and users are unable to connect now. Occasionally someone can connect, but it's after almost 20 minutes of trying.

**Comment by LukasPob** [comment]
yup same issue, this grows into an issue where NOONE is able to connect anymore. Had to wipe the servers

**Comment by karlseven** [comment]
haha and i hope the devs know where to look on this issue. i already put this down in the official discord but no reply
